### PR TITLE
Missing input types search and tel

### PR DESCRIPTION
**Pull request by MidnightDesign**

### CODE REVIEW

File: src/Element/Tel.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+     * @return array
+     */
+    public function getInputSpecification()
+    {
+        return [
+            'name' => $this->getName(),
+            'required' => true,
+            'filters' => [
+                ['name' => StringTrim::class],
+                ['name' => StripNewlines::class],
```
Comment (weierophinney) [inline src/Element/Tel.php]: With this in place, your `RegexValidator` is essentially moot...

Comment (MidnightDesign) [inline src/Element/Tel.php]: Is it? What if someone sets a different input filter?

Comment (weierophinney) [inline src/Element/Tel.php]: If they set their own input filter, they won't get that validator anyways...

Comment (MidnightDesign) [inline src/Element/Tel.php]: No, I mean if they change the filters on the input filter.

File: src/Element/Search.php [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright  Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
```
Comment (samsonasik) [inline src/Element/Search.php]: 2016

**Comment by weierophinney** [comment]
Merged to develop for release with 2.9.0.

**Pull request closed without merging**